pa/fin/tasks.py

```diff
@@ -66,7 +58,19 @@ def update_tickers_statements_task():
     """
     with redis_lock('update_tickers_statements_task') as acquired:
         if acquired:
-            update_tickers_statements()
+            quarter_ago = date.today() - timedelta(30 * 3)
+            tickers_query = Ticker.objects \
```

Comment:
looks like a query without any input params, try to keep such in models in a dedicated QuerySet or Manager